datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx

```diff
@@ -74,12 +66,10 @@ export const DatasetStatsSummary = ({
         !!lastUpdatedMs && (
             <Popover
                 content={
-                    <div>
-                        Changed on {toLocalDateTimeString(lastUpdatedMs)}.{' '}
-                        <Tooltip title="The time at which the data was last changed in the source platform">
-                            <HelpIcon />
-                        </Tooltip>
-                    </div>
+                    <PopoverContent>
+                        The time at which the data was last changed in the source platform:{' '}
```

Comment:
Can we make this more concise? How about:

"Data was last changed in the source platform at 5/8/2021, 07:16PM EDT"

Reply:
i like this better too!

Reply:
maybe change "at" -> "on" -- "Data was last changed in the source platform on 5/8/2021, 07:16PM EDT"